### PR TITLE
Entities list contracts backend

### PR DESCRIPTION
**Pull request by leonprou**
## Proposed changes

Integrating new contracts in the backend flow

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: dapp/src/sagas/directory.js [hunk 1]
```diff
@@ -54,13 +54,17 @@ function * getList ({ tokenAddress }) {
...
+  const CommunityContract = getContract({ abiName: 'Community',
+    address: communityAddress
   })
 
   const { hash } = yield call(createMetadata, { metadata: data })
-  const method = SimpleListContract.methods.addEntity(hash)
+  // TEMP:
+  console.log(hash)
+  const userRoles = '0x0000000000000000000000000000000000000000000000000000000000000001'
+  const method = CommunityContract.methods.addEntity(data.account, userRoles)
```
Comment (leonprou) [inline dapp/src/sagas/directory.js]: @LiorAgnin after the entity is added to the `CommunityContract`. you should send the user metadata to the backend (the backend will put it to the User Registry contract).
the api endpoint is:
```
PUT http://localhost:3000/api/v1/entities/:accountAddress
body:
{
	"metadata": {
		"name": "ha!",
		"type": "user"
                 // ...
	}
}

```

File: server/src/utils/web3.js [hunk 2]
```diff
@@ -17,16 +17,29 @@ const createWeb3 = (providerUrl) => {
...
+  }
+
   const { from } = options
   const gas = await method.estimateGas({ from })
+  const gasPrice = bridgeType === 'home' ? '1000000000' : undefined
   const account = await Account.findOneOrCreate({ bridgeType, address: from })
   let receipt
   try {
-    receipt = await method.send({ ...options, gas, nonce: account.nonce })
+    // console.log(`[${bridgeType}] sending method ${method._method.name} from ${from} with nonce ${account.nonce}. gas price: ${gasPrice}, gas limit: ${gas}`)
```
Comment (LiorAgnin) [inline server/src/utils/web3.js]: clear comments

Comment (leonprou) [reply server/src/utils/web3.js]: done. update your branch :)

**Comment by leonprou** [comment]
merged in https://github.com/ColuLocalNetwork/CLN-community-app/pull/249

**Pull request closed without merging**